include/eve/conditional.hpp

```diff
@@ -99,10 +100,13 @@ namespace eve
       return 0ULL;
     }
 
+    constexpr auto friend operator<=>(ignore_all_ const&, ignore_all_ const&) = default;
```

Comment:
doesnt this give conditional < and > ? I fear they are nto what we think they should be ?